index.html

```diff
@@ -321,25 +324,79 @@ <h1 class="contact-head-me">Contact me</h1>
             </div>
-            <form action="https://formspree.io/f/xnqrezql" method="post" >
+            <form action="https://formspree.io/f/xnqrezql" method="post" class="contact-form" id="form" onload='document.form1.text1.focus()'>
               <ul class="form">
                 <li id="name-text-area ">
-                  <div id="name-basic-container"><input type="text" name="name" placeholder="Enter Full Name" maxlength="30" class= "name-text-input" required />
+                  <div id="name-basic-container"><input type="text" name="name" id="form_input1" placeholder="Enter Full Name" maxlength="30" class= "name-text-input" required />
                   </div>
                 </li>
                 <li id="mail-text-area">
-                  <div id="mail-basic-container"><input type="email" name="e-mail" placeholder="Enter e-mail" class="mail-text-input " required /></div>
+                  <div id="mail-basic-container"><input type="email" name="e-mail" id="form_input2" placeholder="Enter e-mail" class="mail-text-input " required /></div>
                 </li>
                 <li id="text-area">
                   <textarea name="text-area" placeholder="Write your message here" cols="30" rows="10" maxlength="500" class="txt-area txt-cont-out" required></textarea>
                 </li>
               <li class="action">
-                <button type="submit" class="btn-sz-sub sub_text">Get In Touch</button>
+                <button type="submit" id="validate" class="btn-sz-sub sub_text">Get In Touch</button>
               </li>
               </ul>
             </form>
           </div>
       </section>
+
+<section class="ddd">
+  
+  <!--
+      <div class="modal_overdlay">
+        <div id="myModal" class="modal">
+          <div class="modal_pry_txt">
+            <div class="modal_frame">
+              <h1 class="modal_proj_title">Tonic</h1>
+              <span class="modalBtn"><i class=" modal_hamburger fa-solid fa-x"></i></span>
+            </div>
+            <ul class="modal_frame_dev">
+              <li class="modal_tag_list"><h2 class="modal_canopy">CANOPY</h2></li>
+              <span class="modal_counter"></span>
+              <li class="modal_tag_role"><h2 class="modal_role">Back End Dev</h2></li>
+              <span class="modal_counter_tw"></span>
+              <li class="modal_tag_year"><h2 class="modal_yr">2015</h2></li>
+            </ul>
+          </div>
+          <img id="snapsht_avocado" alt="snapshort img" src="./Scr/images/avocado.png" >
+          <div class="modal_Left_Block">
+            <p class="modal_description">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
```

Comment:
- [ ] Please remove all forms of commented code from your codebase as it pollutes it